### ISSUE TITLE
Rasta does tree-shaking it should not do

### ISSUE THREAD

**Issue by mraleph** [dart-lang/rasta#84]
If I compile something like this with rasta to kernel then contents of library `b.dart` are not included into the resulting Kernel file. 

```dart
import 'b.dart';

main() {
}
```

`b.dart`
```dart
library b;

class Bar {
  void foo() { }
}
```

**Comment by mraleph** [comment]
@kmillikin discovered that rasta also tree-shakes parts of core libraries - which blocks his work on switching bootstrapping process to use kernel instead of sources.

**Comment by peter-ahe-google** [comment]
Proposed fix in [CL 2203313002](https://codereview.chromium.org/2203313002).
